src/browser/provider/built-in/chrome/cdp.js

```diff
@@ -0,0 +1,110 @@
+import remoteChrome from 'chrome-remote-interface';
+import fs from 'fs';
+import Promise from 'pinkie';
+import pify from 'pify';
+
+
+const fsWriteFile = pify(fs.writeFile, Promise);
+
+
+async function getActiveTab (cdpPort, browserId) {
+    var tabs = await remoteChrome.listTabs({ port: cdpPort });
+    var tab  = tabs.filter(t => t.type === 'page' && t.url.indexOf(browserId) > -1)[0];
+
+    return tab;
+}
+
+async function getWindowId (client, tab) {
+    try {
+        var { windowId } = await client.Browser.getWindowForTarget({ targetId: tab.id });
+
+        return windowId;
+    }
+    catch (e) {
+        return null;
+    }
+}
+
+async function setEmulationBounds (client, device, overrideWidth, overrideHeight) {
```

Comment:
`overrideWidth` and `overrideHeight` is a new bounds.
So, I propose to pack their to the single argument.
`async function setEmulationBounds (client, device, {newWidth, newHeight})`